src/point-cloud-octree.ts

```diff
@@ -169,7 +171,7 @@ export class PointCloudOctree extends PointCloudTree {
 
       //Initialise the splats mesh if the nodes contain splats information
       if (this.renderAsSplats) {
-        this.splatsMesh.initialize(this.potree.pointBudget, this.loadHarmonics);
+        this.splatsMesh.initialize(MAX_SPLATS_RENDERED, this.loadHarmonics);
```

Comment:
I'm not sure about this change. What about:

```suggestion
        this.splatsMesh.initialize(this.potree.pointBudget ?? MAX_SPLATS_RENDERED, this.loadHarmonics);
```

I'm not sure if point budget can be undefined.. but that way at least you let the user choose the initial state.